test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -61,14 +63,14 @@ def run_model_test(self, model, batch_size=2, state_dict=None,
 
         # export the model to ONNX
         f = io.BytesIO()
-        torch.onnx.export(model, input, f,
-                          opset_version=self.opset_version,
-                          example_outputs=output,
-                          do_constant_folding=do_constant_folding,
-                          keep_initializers_as_inputs=self.keep_initializers_as_inputs,
-                          dynamic_axes=dynamic_axes,
-                          input_names=input_names, output_names=output_names)
-
+        torch.onnx._export(model, input, f,
```

Comment:
so we didn't expose  `fixed_batch_size` to torch.onnx.export?

Reply:
I did not, but if you think we should I could add it.

Reply:
Let's do it in another PR then